_posts/2018-12-20-10-cool-ai-examples.md

```diff
@@ -0,0 +1,197 @@
+
+
+### Example 5 - Nvidia AI based image restoration
+
+<div class="responsive-video">
+    <iframe width="1164" height="655" src="https://www.youtube.com/embed/P0fMwA3X5KI" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
+</div>
+Like the example above this is about denoising but is similar yet different.
+This AI has been trained without ever been shown what noise is, so no before vs after comparison.
+It can remove noise from images, restore images that are almost only visible noise and even remove lots of text from a given image.
+This technique will make cleaning up images much easier and allow us to preserve and restore imagery that might otherwise be lost or unusable!
+
+
+### Example 6 - Noise reduction for path traced GI
+
+<div class="responsive-video">
+    <iframe width="1164" height="655" src="https://www.youtube.com/embed/HSmm_vEVs10" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
+</div>
+This video shows that denoising techniques can have other great benefits in the visual/gaming industry.
+Path traced global illumination (casting light in a 3D scene to determine lighting from a global source like the sun) is a very resource intensive task.
+Current solutions use all sorts of tricks to mimic this but they are not the real deal.
+This technique allows for path traced GI with a very low sample count and denoises the output whilst being temporally stable.
+Something like the [new cards from Nvidia](https://www.youtube.com/watch?v=Ms7d-3Dprio) are now capable of!
+
+
+### Example 7 - Isolate speech signals
+
+<div class="responsive-video">
+    <iframe width="1164" height="655" src="https://www.youtube.com/embed/zL6ltnSKf9k" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
+</div>
+Having an audio or video file with multiple people speaking at once or when there is a lot of background noise can be annoying for various reasons.
+It makes it harder to understand any of the speaking parties.
+This technique allows each speaker's audio to be isolated an listened to without hearing the other sources of interference.
+It is helpful to clean up conversations or remove background noise.
+
+
+### Example 8 - Impersonate anyone
+
+<div class="responsive-video">
+    <iframe width="1164" height="655" src="https://www.youtube.com/embed/EQX1wsL2TSs" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
+</div>
+Soon you will not be able to tell that what you see is actually what happened.
+This advanced technique improves on older versions, and allows you to transfer your facial and torso movements onto a target.
+Techniques like this make it clear that fake news and fake sources of media will become an even bigger problem in the future as this technology becomes even better.
+It might not be such a bad idea to invest in that blockchain backed media repository after all so the validity of media file can be tracked...
```

Comment:
```suggestion
It might not be such a bad idea to invest in that blockchain backed media repository after all so the validity of media files can be tracked...
```